test/extended/util/prometheus/helpers.go

```diff
@@ -171,7 +174,11 @@ func RunQueries(promQueries map[string]bool, oc *exutil.CLI, ns, execPodName, ba
 	if len(queryErrors) != 0 {
 		exutil.DumpPodLogsStartingWith("prometheus-0", oc)
 	}
-	o.Expect(queryErrors).To(o.BeEmpty())
+	var errs []error
+	for _, err := range queryErrors {
+		errs = append(errs, err)
```

Comment:
The previous code (`o.Expect(queryErrors).To(o.BeEmpty())`) was outputting the `query => error` map, but now we're discarding the query information (the key). Perhaps we should do something like `errs = append(errs, fmt.Errorf("Query %q failed: %v", query, err))`.